src/esp/sensor/Sensor.h

```diff
@@ -44,17 +44,14 @@ enum class SensorSubType {
 
 // Specifies the configuration parameters of a sensor
 struct SensorSpec {
-  std::string uuid = "rgba_camera";
-  SensorType sensorType = SensorType::Color;
-  SensorSubType sensorSubType = SensorSubType::Pinhole;
-  std::map<std::string, std::string> parameters = {{"near", "0.01"},
-                                                   {"far", "1000"},
-                                                   {"hfov", "90"},
-                                                   {"ortho_scale", ".1"}};
-  vec3f position = {0, 1.5, 0};
-  vec3f orientation = {0, 0, 0};
-  vec2i resolution = {84, 84};
-  int channels = 4;
+  std::string uuid;
+  SensorType sensorType;
+  SensorSubType sensorSubType;
+  vec3f position;
+  vec3f orientation;
+  vec2i resolution;
```

Comment:
this should go to visual sensor for now.
If non-visual sensor also need it, we can always lift it up later.

Reply:
Ok, I can do that! My reasoning was that position, orientation, and resolution would matter for force and acoustic sensors as well.

Reply:
My comment was on the resolution, not on position or orientation.